tests/test1.sh

```diff
@@ -13,6 +13,7 @@ if [[ "$output" == *"Admiral"* ]]; then
   echo "PASS"
   exit 0
 else
-  echo "FAIL" . $output
```

Comment:
Why is this change needed?

Reply:
I rechecked. I'm very sure FAIL didn't get printed and I'm not very sure why... so that' why I changed it to this.